app/models/concerns/rubygem_searchable.rb

```diff
@@ -46,6 +47,7 @@ def as_indexed_json(_options = {}) # rubocop:disable Metrics/MethodLength
         }
       }.merge!(suggest_json)
     end
+    # rubocop:enable Metrics/CyclomaticComplexity
```

Comment:
Why do we need the todo and enable line?

Reply:
When I regenerated the rubocop todo file, I ran into this violation and ran `--disable-uncorrectable` to add those comments. Now I realized that the exclude files from `Metrics/CyclomaticComplexity:` was removed for some reason and I manually added it back in.

```
Exclude:
    - 'app/models/concerns/rubygem_searchable.rb'
```